install_config/certificate_customization.adoc

```diff
@@ -95,3 +95,6 @@ Relative paths are resolved relative to the master configuration file. Restart
 the server to pick up the configuration changes.
 
 For the master API or web console, wildcard names are accepted.
+
+The provided named certificate can also be added using the 
+xref:../install_config/redeploying_certificates.adoc#redeploying-new-custom-ca[*_redeploy-openshift-ca.yml_* playbook]. 
```

Comment:
hi, the playbook used for adding or updating named certificate should be `redeploy-certificates.yml` instead of `redeploy-openshift-ca.yml`   https://github.com/mburke5678/openshift-docs/blob/56da030877d899a34061116843c4177e97c0e785/install_config/redeploying_certificates.adoc#redeploying-all-certificates-using-the-current-product-title-and-etcd-ca